internal/pkg/telemetry/ddtracer_test.go

```diff
@@ -21,39 +23,44 @@ func TestContextAttributes(t *testing.T) {
 	mockTracer := mocktracer.Start()
 	defer mockTracer.Stop()
 
-	logger := log.NewDebugLogger()
-
-	proc, err := servicectx.New(servicectx.WithLogger(logger), servicectx.WithUniqueID("mockid"))
-	require.NoError(t, err)
-
 	// Setup telemetry
-	tel, err := New(
+	logger := log.NewDebugLogger()
+	tel, err := telemetry.New(
 		func() (trace.TracerProvider, error) {
-			return NewDDTracerProvider(logger, proc), nil
+			return telemetry.NewDDTracerProvider(logger, servicectx.NewForTest(t)), nil
 		},
 		nil,
 	)
 	require.NoError(t, err)
 
+	// Add some common context attribute, it should appear in span and log record
 	ctx := context.Background()
+	ctx = ctxattr.ContextWith(ctx, attribute.String("foo", "bar"))
 
+	// Create span
 	ctx, span := tel.Tracer().Start(ctx, "keboola.go.test")
 	defer span.End(nil)
 
+	// Tracer added TraceID and SpanID attributes to the context
```

Comment:
BTW: I know that I use comments more than is standard. But otherwise I can't quickly scan a code longer than 20 lines.